drivers/sensor/iis2dlpc/iis2dlpc.c

```diff
@@ -380,21 +393,15 @@ const struct iis2dlpc_device_config iis2dlpc_cfg = {
 	.int_gpio_port = DT_INST_GPIO_LABEL(0, drdy_gpios),
 	.int_gpio_pin = DT_INST_GPIO_PIN(0, drdy_gpios),
 	.int_gpio_flags = DT_INST_GPIO_FLAGS(0, drdy_gpios),
-#ifdef CONFIG_IIS2DLPC_PULSE
-#if defined(CONFIG_IIS2DLPC_ONLY_SINGLE)
-	.pulse_trigger = IIS2DLPC_ONLY_SINGLE,
-#elif defined(CONFIG_IIS2DLPC_SINGLE_DOUBLE)
-	.pulse_trigger = IIS2DLPC_BOTH_SINGLE_DOUBLE,
-#endif
-	.pulse_ths[0] = CONFIG_IIS2DLPC_PULSE_THSX,
-	.pulse_ths[1] = CONFIG_IIS2DLPC_PULSE_THSY,
-	.pulse_ths[2] = CONFIG_IIS2DLPC_PULSE_THSZ,
-	.pulse_shock = CONFIG_IIS2DLPC_PULSE_SHOCK,
-	.pulse_ltncy = CONFIG_IIS2DLPC_PULSE_LTNCY,
-	.pulse_quiet = CONFIG_IIS2DLPC_PULSE_QUIET,
-#endif /* CONFIG_IIS2DLPC_PULSE */
 	.drdy_int = DT_INST_PROP(0, drdy_int),
 
+#ifdef CONFIG_IIS2DLPC_TAP
+	.tap_mode = DT_INST_PROP(0, tap_mode),
+	.tap_ths = DT_INST_PROP(0, tap_ths),
```

Comment:
This could be `tap_threshold`.

Reply:
sure

Reply:
BTW, this is the name in the datasheet, but there is no reason to keep it same in driver